internal/export/database/export.go

```diff
@@ -425,7 +425,7 @@ func (db *ExportDB) LatestExportBatchEnd(ctx context.Context, ec *model.ExportCo
 // ListLatestExportBatchEnds returns a map of export config IDs to their latest
 // batch end times.
 func (db *ExportDB) ListLatestExportBatchEnds(ctx context.Context) (map[int64]*time.Time, error) {
-	var ts map[int64]*time.Time
+	ts := make(map[int64]*time.Time, 8)
```

Comment:
Where is the number 8 from? Should there be a comment explaining it?

Reply:
It's completely arbitrary. It pre-allocates the map of that size. We have no way of knowing how many records there will be, but 8 is pretty safe"

Reply:
ack